### PR TITLE
DEVREL-1287 - Minor Typographic Changes

### PR DESCRIPTION
**Pull request by ChrissyAtWork**
CordApp -> CorDapp

👮🏻👮🏻👮🏻 !!!! DESCRIBE YOUR CHANGES HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you run the unit, integration and smoke tests as described [here](https://docs.corda.net/head/testing.html)?
- [ ] If you added public APIs, did you write the JavaDocs?
- [ ] If the changes are of interest to application developers, have you added them to the [changelog](https://github.com/corda/corda/blob/master/docs/source/changelog.rst), and potentially the [release notes](https://github.com/corda/corda/blob/master/docs/source/release-notes.rst)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/corda/corda/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/corda/corda/blob/master/CONTRIBUTING.md#developer-certificate-of-origin).

Thanks for your code, it's appreciated! :)

### CODE REVIEW

**Comment by ChrissyAtWork** [comment]
Only the last one has actual change, otherwise it was just that I noticed a few case problems as I went through.

File: docs/source/key-concepts-contracts.rst [hunk 1]
```diff
@@ -3,9 +3,9 @@ Contracts
 
 .. topic:: Summary
 
-   * *A valid transaction must be accepted by the contract of each of its input and output states*
-   * *Contracts are written in a JVM programming language (e.g. Java or Kotlin)*
-   * *Contract execution is deterministic and its acceptance of a transaction is based on the transaction's contents alone*
+   * *A transaction is contractually valid if all of its input and output states are acceptable according to the contract.*
+   * *Contracts are written in a JVM programming language such as Java or Kotlin.*
```
Comment (joeldudleyr3) [inline docs/source/key-concepts-contracts.rst]: We actually only guarantee support for Java and Kotlin. I believe Scala doesn't work, for example. Maybe best to rephrase this to `Contracts are written in Java or Kotlin`?

File: docs/source/key-concepts-contracts.rst [hunk 2]
```diff
@@ -39,11 +39,11 @@ We can picture this situation as follows:
 The contract code can be written in any JVM language, and has access to the full capabilities of the language,
 including:
 
-* Checking the number of inputs, outputs, commands, time-window, and/or attachments
+* Checking the number of inputs, outputs, commands, time-windows or attachments
```
Comment (joeldudleyr3) [inline docs/source/key-concepts-contracts.rst]: So there is either a time-window, or not. Not sure if it's clearer to say, "Checking the number of inputs, outputs, commands or attachments, and whether there is a time-window"?

File: docs/source/key-concepts-contracts.rst [hunk 3]
```diff
@@ -58,11 +58,11 @@ on the network reach consensus regarding the validity of a given ledger update.
 
 Future versions of Corda will evaluate transactions in a strictly deterministic sandbox. The sandbox has a whitelist that
 prevents the contract from importing libraries that could be a source of non-determinism. This includes libraries
-that provide the current time, random number generators, libraries that provide filesystem access or networking
+that provide the current time, random number generators, libraries that provide file system access or networking
 libraries, for example. Ultimately, the only information available to the contract when verifying the transaction is
 the information included in the transaction itself.
 
-Developers can pre-verify their CorDapps are determinsitic by linking their CorDapps against the deterministic modules
+**Tip:** Developers can pre-verify that their CorDapps are determinsitic by linking their CorDapps against the deterministic modules
```
Comment (joeldudleyr3) [inline docs/source/key-concepts-contracts.rst]: Leftover typo in "determinsitic" there :)

File: docs/source/key-concepts-contracts.rst (hunk 1 above)
Comment (ChrissyAtWork) [reply docs/source/key-concepts-contracts.rst]: done

File: docs/source/key-concepts-contracts.rst (hunk 2 above)
Comment (ChrissyAtWork) [reply docs/source/key-concepts-contracts.rst]: done and fixed the reference to any JVM language

**Comment by mikehearn** [comment]
Thanks for the fixes!
